include/Graph/Graph.hpp

```diff
@@ -503,17 +520,15 @@ class Graph {
    * @param workingDir The path to the directory in which is placed the Output
    * file
    * @param OFileName The Output File Name ( )
-   * @param compress Indicates if the Output will be compressed ( Pay Attention
-   * if compress flag is true, not compressed files will be deleted [ #48 ] )
    * @param writeNodeFeat Indicates if export also Node Features
    * @param writeEdgeWeight Indicates if export also Edge Weights
    * @return 0 if all OK, else return a negative value
    */
   virtual int writeToFile(
       InputOutputFormat format = InputOutputFormat::STANDARD_CSV,
       const std::string &workingDir = ".",
-      const std::string &OFileName = "graph", bool compress = false,
-      bool writeNodeFeat = false, bool writeEdgeWeight = false) const;
+      const std::string &OFileName = "graph", bool writeNodeFeat = false,
+      bool writeEdgeWeight = false) const;
```

Comment:
⚠️ Public API change. We're removing the ability to compress. Users can use their own compression wrappers if they want.